src/components/board-feed/board-feed.js

```diff
@@ -0,0 +1,39 @@
+import {BoardCard} from "../board-card/board-card.js";
+
+export class BoardFeed extends HTMLElement {
+
+    constructor() {
+        super();
+        this.pins = []
+    }
+
+    connectedCallback() {
+        fetch('components/board-feed/board-feed.html')
+            .then(response => response.text())
+            .then(html => {            
+                this.innerHTML = html;
+                this.addButtonEvents();
+                this.populateFeed();
+            });
+    }
+
+    attributeChangedCallback(property, oldValue, newValue) {
+        if (oldValue === newValue) return;
+        this[property] = newValue;
+    }
+
+    populateFeed() {
+        customElements.define('board-card', BoardCard);
+    }
```

Comment:
this could be moved to the main script.js. Or else it will be defined by every element that uses it.